content/javascript/concepts/arrays/terms/includes/includes.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: '.includes()'
+Description: 'Returns true if a given value is included in an array.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.includes()` method returns `true` if a given value is included in an array. Otherwise, it returns `false`.
+
+## Syntax
+
+```pseudo
+array.includes(value, index=0);
+```
+
+The following parameters are used:
+
+- A case-sensitive and type-sensitive `value` that is checked for inclusion in the `array`.
+- An optional `index`, defaulted to 0, that tells `.includes()` where to begin the check.
+
+Here are some edge-cases to consider when using `.includes()`:
+
+- It will not work if the provided `index` is greater than the length of the array. Instead, `false` will be returned.
+- If the `index` is less than or equal to 0, the entire array will be searched.
+- By itself, this method isn't suitable for nested array and should be used with the `.flat()` and/or [`.find()`](https://www.codecademy.com/resources/docs/javascript/arrays/find) methods.
+
+For objects, `.includes()` only returns `true` for references of the same object. Even checking against an object literal with the same property-value pairs will yield `false`.
```

Comment:
```suggestion
- For objects, `.includes()` only returns `true` for references of the same object. Even checking against an object literal with the same property-value pairs will yield `false`.
```

Reply:
I wasn't sure whether to also make this a list item since I used two sentences to make this point. But I can try and consolidate with a local change and push later tonight (or early tomorrow morning).